src/Cms/Language.php

```diff
@@ -225,10 +225,10 @@ public static function create(array $props)
 		}
 
 		// update the main languages collection in the app instance
-		App::instance()->languages(false)->append($language->code(), $language);
+		$kirby->languages(false)->append($language->code(), $language);
 
 		// trigger after hook
-		$kirby->trigger('language.create:after', compact('language', 'props'));
+		$kirby->trigger('language.create:after', ['language' => $language, 'input' => $props]));
```

Comment:
Yikes, this happens when I use the code suggestion feature here. There are now additional parenthesis. I guess it's easier to fix this locally.

Reply:
Actually I already fixed. This line should be outdated. Where do you find this?

Reply:
Oh great. Then it was just Github showing the outdated version.